awscli/compat.py

```diff
@@ -26,6 +27,15 @@
 StringIO = six.StringIO
 urlopen = six.moves.urllib.request.urlopen
 
+# Most, but not all, python installations will have zlib. This is required to
+# compress any files we send via a push. If we can't compress, we can still
+# package the files in a zip container.
+try:
+    import zlib
+    COMPRESSION_MODE = zipfile.ZIP_DEFLATED
```

Comment:
Can we rename it to something specific to zipfile's like ``ZIP_COMPRESSION_MODE`` to be more explicit as it is located in compat.py module which is shared all across the CLI codebase?